benchmarks/data/.keep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Git can't track empty directories, the `.keep` is here to make sure the directory exists